src/core/viz/expressions/torque.js

```diff
@@ -143,11 +143,15 @@ export class Torque extends BaseExpression {
     }
     eval(feature) {
         const input = this.input.eval(feature);
+        if (Number.isNaN(input)){
```

Comment:
we should document the behaviour for null/nan values

Reply:
yep